### DISCUSSION TITLE
Is there a way to create a csharp type from string using Jint?

### DISCUSSION THREAD

**Discussion by hyzx86** [sebastienros/jint#1265] [Q&A]
I generate a C # class according to the dynamic type of orchardcore and cooperate, with FreeSQL to generate different table structures, so as to encapsulate changes in different databases

However, FreeSQL needs to specify a predefined type
Is there any way to register this type to the current context through jint?
![image](https://user-images.githubusercontent.com/15613121/186456881-1e047df0-64c5-4823-80fa-866251d9f59e.png)

**Comment by lahma** [comment]
Have you tried adding a TypeReference to the engine?

**Reply by hyzx86** [reply]
The current situation is that this type has not been registered in the assembly, and they are only strings at present~
